### PR TITLE
fix(editor): Disable Ask AI button in deprecated nodes (no-changelog)

### PR DESCRIPTION
**Pull request by RicardoE105**
## Summary

Title self explanatory

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/ADO-2437/disable-ask-ai-button-in-old-function-node

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/components/Error/NodeErrorView.vue [hunk 1]
```diff
@@ -428,7 +433,7 @@ async function onAskAssistantClick() {
 				v-n8n-html="getErrorDescription()"
 			></div>
 			<div
-				v-if="isAskAssistantAvailable"
+				v-if="isAskAssistantAvailable && !nodeIsHidden()"
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/components/Error/NodeErrorView.vue:436]: I think this should all be part of `isAskAssistantAvailable` so we don't add extra logic to Vue templates.

**Review by MiloradFilipovic — DISMISSED** [review]
Added minor comment but looks good otherwise.

File: packages/editor-ui/src/components/Error/NodeErrorView.vue (hunk 1 above)
Comment (RicardoE105) [reply packages/editor-ui/src/components/Error/NodeErrorView.vue]: Good idea 🙏

**Review by MiloradFilipovic — APPROVED** [review]
LGTM

**Comment by janober** [comment]
Got released with [`n8n@1.64.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.64.0)

**Pull request merged**